backoff.go

```diff
@@ -10,7 +10,7 @@ import (
 var (
 	DefaultBackoffConfig = BackoffConfig{
 		MaxDelay:  120 * time.Second,
-		baseDelay: 1.0 * time.Second,
+		BaseDelay: 1.0 * time.Second,
```

Comment:
The test needs to sleep long enough to allow the transport to reconnect - the backoff is controlled by this value, so the tests needs to be able to inspect (and modify for speed) this avlue.